src/scroll_list/ScrollList.js

```diff
@@ -1066,6 +1066,7 @@ define(function(require) {
                     panToOptions,
                     offset,
                     itemLayout.scaleToFit,
+                    (itemLayout.left || 0 + itemLayout.paddingLeft || 0),
```

Comment:
I had to add the `|| 0` to each statement as a guard because in cases where one or both of those properties are undefined, a NaN gets propagated around for the x-coordinate causing bad stuff to happen.

I agree that it looks kind of goofy, but i dont have any great way of improving it besides adding ()'s around the individual expressions, `((itemLayout.left || 0) + (itemLayout.paddingLeft || 0))`which doesnt look much better.